actix-server/src/server.rs

```diff
@@ -193,14 +193,14 @@ impl ServerInner {
         let is_tokio = tokio::runtime::Handle::try_current().is_ok();
 
         match (is_actix, is_tokio) {
-            (true, _) => info!("Actix runtime found; starting in Actix runtime"),
-            (_, true) => info!("Tokio runtime found; starting in existing Tokio runtime"),
+            (true, _) => info!("actix runtime found; starting in Actix runtime"),
+            (_, true) => info!("tokio runtime found; starting in existing Tokio runtime"),
```

Comment:
Capitalization here is inconsistent, mixed use of actix/Actix and tokio/Tokio.

Reply:
fixed